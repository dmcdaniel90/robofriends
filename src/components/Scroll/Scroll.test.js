import {render} from '@testing-library/react'
import Scroll from './Scroll';

describe('Scroll', () => {
  it('expect to render Scroll component', () => {
    const {container} = render(<Scroll />)
    expect(container).toMatchSnapshot();
  })
})