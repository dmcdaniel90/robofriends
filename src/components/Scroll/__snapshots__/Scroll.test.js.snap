// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll expect to render Scroll component 1`] = `
<div>
  <div
    style="overflow-y: scroll; border: 3px solid black; height: 800px;"
  />
</div>
`;
